.mkrepo/main.w

```diff
@@ -29,7 +29,11 @@ for file in fs.readdir(".") {
   libs.push(lib);
 }
 
-readme.update(libs.copy());
+try {
+  readme.update(libs.copy());
```

Comment:
Why is this needed?

Reply:
I was trying it out cause I'm getting this error on wingcloud

<img width="869" alt="image" src="https://github.com/winglang/winglibs/assets/5547636/7c42c7a2-eff6-4aca-8f4f-35a03c85bc45">

https://wing.cloud/polamoros/winglibs/logs/main